LaserTrajCmd command
---
time start_time